test/client/js/libs/handlebars/parse.js

```diff
@@ -358,16 +358,16 @@ describe("parse Handlebars helper", () => {
 	[{
 		name: "in text",
 		input: "Hello💬",
-		expected: 'Hello<span class="emoji">💬</span>',
+		expected: 'Hello<span class="emoji" role="img" aria-label="Emoji: speech_balloon" title="speech_balloon">💬</span>',
```

Comment:
After you use `name` instead of `shortname`, this will automatically become `"Emoji: speech balloon"`, which is much better read by screen reader (it also has better description in some cases).

Reply:
Done